public/app/core/components/Select/FolderPicker.tsx

```diff
@@ -14,6 +14,7 @@ export interface Props {
   dashboardId?: any;
   initialTitle?: string;
   initialFolderId?: number;
+  useInNextGenForms?: boolean;
```

Comment:
We use `useNewForms`. I've made similar changes to folder picker in my Dashboard Save Modals PR